[CI] DocsClientYamlTestSuiteIT test {yaml=reference/rest-api/usage/line_38} failing

It looks like there is some issue with the response format. 

**Build scan:**
https://gradle-enterprise.elastic.co/s/fgywlwd2jzobe/tests/:docs:yamlRestTest/org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT/test%20%7Byaml=reference%2Frest-api%2Fusage%2Fline_38%7D

**Reproduction line:**
```
./gradlew ':docs:yamlRestTest' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/rest-api/usage/line_38}" -Dtests.seed=A2E9F58B9E81A215 -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dtests.locale=zh-CN -Dtests.timezone=Etc/UTC -Druntime.java=20
```

**Applicable branches:**
main

**Reproduces locally?:**
Yes

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT&tests.test=test%20%7Byaml%3Dreference/rest-api/usage/line_38%7D

**Failure excerpt:**
```
java.lang.AssertionError: Failure at [reference/rest-api/usage:16]: Expected a map containing
                         eql: a map containing
                     available: <true>
                       enabled: <true>
                      features: a map containing
                           joins: a map containing
                join_queries_three: <0>
                  join_queries_two: <0>
                        join_until: <0>
         join_queries_five_or_more: <0>
                 join_queries_four: <0>
                        sequence: <4>
                            keys: a map containing
                     join_keys_two: <0>
                     join_keys_one: <3>
                   join_keys_three: <0>
            join_keys_five_or_more: <0>
                    join_keys_four: <0>
                            join: <0>
                       sequences: a map containing
            sequence_queries_three: <1>
             sequence_queries_four: <0>
              sequence_queries_two: <3>
                    sequence_until: <0>
                  sequence_maxspan: <2>
     sequence_queries_five_or_more: <0>
                           event: <8>
                           pipes: a map containing
                         pipe_tail: <9>
                         pipe_head: <0>
                       queries: a map containing
                             all: a map containing
                             total: <12>
                            failed: <0>
                            _all: a map containing
                             total: <12>
                            failed: <0>
              frozen_indices: a map containing
                 indices_count: <0>
                     available: <true>
                       enabled: <true>
                         sql: a map containing
                      features: a map containing
                          having: <1>
                       subselect: <0>
                           limit: <1>
                         orderby: <7>
                           where: <3>
                            join: <0>
                         groupby: <1>
                         command: <0>
                           local: <0>
                     available: <true>
                       queries: a map containing
                            rest: a map containing
                             total: <10>
                            paging: <1>
                            failed: <0>
                             cli: a map containing
                             total: <0>
                            paging: <0>
                            failed: <0>
                          canvas: a map containing
                             total: <0>
                            paging: <0>
                            failed: <0>
                            odbc: a map containing
                             total: <0>
                            paging: <0>
                            failed: <0>
                            jdbc: a map containing
                             total: <0>
                            paging: <0>
                            failed: <0>
                          odbc32: a map containing
                             total: <0>
                            paging: <0>
                            failed: <0>
                          odbc64: a map containing
                             total: <0>
                            paging: <0>
                            failed: <0>
                            _all: a map containing
                             total: <10>
                            paging: <1>
                            failed: <0>
                       translate: a map containing
                             count: <1>
                       enabled: <true>
                      rollup: a map containing
                     available: <true>
                       enabled: <true>
        searchable_snapshots: a map containing
    shared_cache_indices_count: <0>
                 indices_count: <0>
                     available: <true>
       full_copy_indices_count: <0>
                       enabled: <true>
                   analytics: a map containing
                         stats: a map containing
                      rate_usage: <2>
                 normalize_usage: <0>
                   boxplot_usage: <3>
               top_metrics_usage: <4>
               multi_terms_usage: <1>
    cumulative_cardinality_usage: <0>
              string_stats_usage: <1>
        moving_percentiles_usage: <0>
                    t_test_usage: <1>
                     available: <true>
                       enabled: <true>
                    logstash: a map containing
                     available: <true>
                       enabled: <true>
                    security: a map containing
                     available: <true>
                       enabled: <false>
                   transform: a map containing
                     available: <true>
                       enabled: <true>
                         ccr: a map containing
    auto_follow_patterns_count: <0>
                     available: <true>
        follower_indices_count: <0>
                       enabled: <true>
                  health_api: a map containing
                     available: <true>
                       enabled: <true>
                      statuses: a map containing
                           green: <5>
                          values: a list containing
                                 0: "green"
                   invocations: a map containing
                           total: <5>
                    verbose_true: <5>
                     spatial: a map containing
                     available: <true>
                       enabled: <true>
                          ml: a map containing
                     inference: a map containing
                     deployments: a map containing
                           time_ms: a map containing
                                 avg: <0.0>
                             count: <0>
                 model_sizes_bytes: a map containing
                               total: <0.0>
                                 min: <0.0>
                                 avg: <0.0>
                                 max: <0.0>
                  inference_counts: a map containing
                               total: <0.0>
                                 min: <0.0>
                                 avg: <0.0>
                                 max: <0.0>
                  trained_models: a map containing
                  model_size_bytes: a map containing
                               total: <0.0>
                                 min: <0.0>
                                 avg: <0.0>
                                 max: <0.0>
              estimated_operations: a map containing
                               total: <0.0>
                                 min: <0.0>
                                 avg: <0.0>
                                 max: <0.0>
                             count: a map containing
                               total: <1>
                         prepackaged: <1>
                               other: <0>
                              _all: a map containing
                               count: <1>
               ingest_processors: a map containing
                              _all: a map containing
                           pipelines: a map containing
                                 count: <0>
                             time_ms: a map containing
                                   min: <0>
                                   max: <0>
                                   sum: <0>
                  num_docs_processed: a map containing
                                   min: <0>
                                   max: <0>
                                   sum: <0>
                        num_failures: a map containing
                                   min: <0>
                                   max: <0>
                                   sum: <0>
     data_frame_analytics_jobs: a map containing
                    memory_usage: a map containing
                  peak_usage_bytes: a map containing
                               total: <0.0>
                                 min: <0.0>
                                 avg: <0.0>
                                 max: <0.0>
                            _all: a map containing
                             count: <0>
                 analysis_counts: an empty map
                          jobs: a map containing
                            _all: a map containing
                             count: <0>
                         detectors: a map containing
                               total: <0.0>
                                 min: <0.0>
                                 avg: <0.0>
                                 max: <0.0>
                        created_by: an empty map
                        model_size: a map containing
                               total: <0.0>
                                 min: <0.0>
                                 avg: <0.0>
                                 max: <0.0>
                         forecasts: a map containing
                               total: <0>
                     forecasted_jobs: <0>
                     available: <true>
                     datafeeds: a map containing
                            _all: a map containing
                             count: <0>
                    node_count: <1>
                       enabled: <true>
                     watcher: a map containing
                     execution: a map containing
                         actions: a map containing
                              _all: a map containing
                               total: <0>
                    total_time_in_ms: <0>
                         watch: a map containing
                           input: a map containing
                              _all: a map containing
                               total: <0>
                              active: <0>
                         trigger: a map containing
                              _all: a map containing
                               total: <0>
                              active: <0>
                     available: <true>
                         count: a map containing
                           total: <0>
                          active: <0>
                       enabled: <true>
           enterprise_search: a map containing
           search_applications: a map containing
                           count: <0>
         analytics_collections: a map containing
                           count: <0>
                     available: <true>
                       enabled: <true>
                     archive: a map containing
                 indices_count: <0>
                     available: <true>
                       enabled: <true>
                         ilm: a map containing
                  policy_count: <17>
                  policy_stats: a list containing
                               0: a map containing
                            phases: a map containing
                                warm: a map containing
                               min_age: <172800000>
                               actions: a list containing
                                       0: "shrink"
                                       1: "forcemerge"
                                       2: "migrate"
                        configurations: a map containing
                              forcemerge: a map containing
                          max_num_segments: <1>
                                  shrink: a map containing
                          number_of_shards: <1>
                                cold: a map containing
                               min_age: <2592000000L>
                               actions: a list containing
                                       0: "migrate"
                        configurations: an empty map
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "30d"
                            max_age_millis: <2592000000L>
                    max_primary_shard_size: "50gb"
              max_primary_shard_size_bytes: <53687091200L>
                              delete: a map containing
                               min_age: <7776000000L>
                               actions: a list containing
                                       0: "delete"
                        configurations: an empty map
                   indices_managed: <0>
                               1: a map containing
                            phases: a map containing
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "3d"
                            max_age_millis: <259200000>
                    max_primary_shard_size: "50gb"
              max_primary_shard_size_bytes: <53687091200L>
                              delete: a map containing
                               min_age: <345600000>
                               actions: a list containing
                                       0: "delete"
                        configurations: an empty map
                   indices_managed: <0>
                               2: a map containing
                            phases: a map containing
                                warm: a map containing
                               min_age: <172800000>
                               actions: a list containing
                                       0: "shrink"
                                       1: "forcemerge"
                                       2: "migrate"
                        configurations: a map containing
                              forcemerge: a map containing
                          max_num_segments: <1>
                                  shrink: a map containing
                          number_of_shards: <1>
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "30d"
                            max_age_millis: <2592000000L>
                    max_primary_shard_size: "50gb"
              max_primary_shard_size_bytes: <53687091200L>
                              delete: a map containing
                               min_age: <2592000000L>
                               actions: a list containing
                                       0: "delete"
                        configurations: an empty map
                   indices_managed: <0>
                               3: a map containing
                            phases: a map containing
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "30d"
                            max_age_millis: <2592000000L>
                    max_primary_shard_size: "50gb"
              max_primary_shard_size_bytes: <53687091200L>
                   indices_managed: <0>
                               4: a map containing
                            phases: a map containing
                                warm: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "forcemerge"
                                       1: "migrate"
                        configurations: a map containing
                              forcemerge: a map containing
                          max_num_segments: <1>
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "3d"
                            max_age_millis: <259200000>
                    max_primary_shard_size: "50gb"
              max_primary_shard_size_bytes: <53687091200L>
                              delete: a map containing
                               min_age: <259200000>
                               actions: a list containing
                                       0: "delete"
                        configurations: an empty map
                   indices_managed: <0>
                               5: a map containing
                            phases: a map containing
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "30d"
                            max_age_millis: <2592000000L>
                    max_primary_shard_size: "50gb"
              max_primary_shard_size_bytes: <53687091200L>
                              delete: a map containing
                               min_age: <7776000000L>
                               actions: a list containing
                                       0: "delete"
                        configurations: an empty map
                   indices_managed: <0>
                               6: a map containing
                            phases: a map containing
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "30d"
                            max_age_millis: <2592000000L>
                                  max_size: "300gb"
                            max_size_bytes: <322122547200L>
                              delete: a map containing
                               min_age: <7776000000L>
                               actions: a list containing
                                       0: "delete"
                        configurations: an empty map
                   indices_managed: <0>
                               7: a map containing
                            phases: a map containing
                                warm: a map containing
                               min_age: <172800000>
                               actions: a list containing
                                       0: "shrink"
                                       1: "forcemerge"
                                       2: "migrate"
                        configurations: a map containing
                              forcemerge: a map containing
                          max_num_segments: <1>
                                  shrink: a map containing
                          number_of_shards: <1>
                                cold: a map containing
                               min_age: <2592000000L>
                               actions: a list containing
                                       0: "migrate"
                        configurations: an empty map
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "30d"
                            max_age_millis: <2592000000L>
                    max_primary_shard_size: "50gb"
              max_primary_shard_size_bytes: <53687091200L>
                              delete: a map containing
                               min_age: <31536000000L>
                               actions: a list containing
                                       0: "delete"
                        configurations: an empty map
                   indices_managed: <0>
                               8: a map containing
                            phases: a map containing
                                warm: a map containing
                               min_age: <172800000>
                               actions: a list containing
                                       0: "shrink"
                                       1: "forcemerge"
                                       2: "migrate"
                        configurations: a map containing
                              forcemerge: a map containing
                          max_num_segments: <1>
                                  shrink: a map containing
                          number_of_shards: <1>
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "7d"
                            max_age_millis: <604800000>
                    max_primary_shard_size: "50gb"
              max_primary_shard_size_bytes: <53687091200L>
                              delete: a map containing
                               min_age: <604800000>
                               actions: a list containing
                                       0: "delete"
                        configurations: an empty map
                   indices_managed: <0>
                               9: a map containing
                            phases: a map containing
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "30d"
                            max_age_millis: <2592000000L>
                    max_primary_shard_size: "10gb"
              max_primary_shard_size_bytes: <10737418240L>
                              delete: a map containing
                               min_age: <2592000000L>
                               actions: a list containing
                                       0: "delete"
                        configurations: an empty map
                   indices_managed: <0>
                              10: a map containing
                            phases: a map containing
                                warm: a map containing
                               min_age: <172800000>
                               actions: a list containing
                                       0: "shrink"
                                       1: "forcemerge"
                                       2: "migrate"
                        configurations: a map containing
                              forcemerge: a map containing
                          max_num_segments: <1>
                                  shrink: a map containing
                          number_of_shards: <1>
                                cold: a map containing
                               min_age: <2592000000L>
                               actions: a list containing
                                       0: "migrate"
                        configurations: an empty map
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "30d"
                            max_age_millis: <2592000000L>
                    max_primary_shard_size: "50gb"
              max_primary_shard_size_bytes: <53687091200L>
                              delete: a map containing
                               min_age: <15552000000L>
                               actions: a list containing
                                       0: "delete"
                        configurations: an empty map
                   indices_managed: <0>
                              11: a map containing
                            phases: a map containing
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "30d"
                            max_age_millis: <2592000000L>
                    max_primary_shard_size: "50gb"
              max_primary_shard_size_bytes: <53687091200L>
                   indices_managed: <0>
                              12: a map containing
                            phases: a map containing
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                    max_primary_shard_size: "50gb"
              max_primary_shard_size_bytes: <53687091200L>
                   indices_managed: <0>
                              13: a map containing
                            phases: a map containing
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "7d"
                            max_age_millis: <604800000>
                                  max_size: "10gb"
                            max_size_bytes: <10737418240L>
                              delete: a map containing
                               min_age: <604800000>
                               actions: a list containing
                                       0: "delete"
                        configurations: an empty map
                   indices_managed: <0>
                              14: a map containing
                            phases: a map containing
                                warm: a map containing
                               min_age: <172800000>
                               actions: a list containing
                                       0: "shrink"
                                       1: "forcemerge"
                                       2: "migrate"
                        configurations: a map containing
                              forcemerge: a map containing
                          max_num_segments: <1>
                                  shrink: a map containing
                          number_of_shards: <1>
                                cold: a map containing
                               min_age: <2592000000L>
                               actions: a list containing
                                       0: "migrate"
                        configurations: an empty map
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "30d"
                            max_age_millis: <2592000000L>
                    max_primary_shard_size: "50gb"
              max_primary_shard_size_bytes: <53687091200L>
                              delete: a map containing
                               min_age: <15552000000L>
                               actions: a list containing
                                       0: "delete"
                        configurations: an empty map
                   indices_managed: <0>
                              15: a map containing
                            phases: a map containing
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "30d"
                            max_age_millis: <2592000000L>
                    max_primary_shard_size: "50gb"
              max_primary_shard_size_bytes: <53687091200L>
                   indices_managed: <0>
                              16: a map containing
                            phases: a map containing
                                 hot: a map containing
                               min_age: <0>
                               actions: a list containing
                                       0: "rollover"
                        configurations: a map containing
                                rollover: a map containing
                                   max_age: "30d"
                            max_age_millis: <2592000000L>
                                  max_size: "10gb"
                            max_size_bytes: <10737418240L>
                              delete: a map containing
                               min_age: <7776000000L>
                               actions: a list containing
                                       0: "delete"
                        configurations: an empty map
                   indices_managed: <0>
                  monitoring: a map containing
            collection_enabled: <false>
                     available: <true>
             enabled_exporters: a map containing
                           local: <1>
                       enabled: <true>
                       graph: a map containing
                     available: <true>
                       enabled: <true>
            aggregate_metric: a map containing
                     available: <true>
                       enabled: <true>
                  data_tiers: a map containing
                     data_warm: a map containing
 primary_shard_size_median_bytes: <0>
             primary_shard_count: <0>
                       doc_count: <0>
    primary_shard_size_avg_bytes: <0>
              primary_size_bytes: <0>
                total_size_bytes: <0>
               total_shard_count: <0>
    primary_shard_size_mad_bytes: <0>
                      node_count: <1>
                     index_count: <0>
                   data_frozen: a map containing
 primary_shard_size_median_bytes: <0>
             primary_shard_count: <0>
                       doc_count: <0>
    primary_shard_size_avg_bytes: <0>
              primary_size_bytes: <0>
                total_size_bytes: <0>
               total_shard_count: <0>
    primary_shard_size_mad_bytes: <0>
                      node_count: <1>
                     index_count: <0>
                     data_cold: a map containing
 primary_shard_size_median_bytes: <0>
             primary_shard_count: <0>
                       doc_count: <0>
    primary_shard_size_avg_bytes: <0>
              primary_size_bytes: <0>
                total_size_bytes: <0>
               total_shard_count: <0>
    primary_shard_size_mad_bytes: <0>
                      node_count: <1>
                     index_count: <0>
                     available: <true>
                  data_content: a map containing
 primary_shard_size_median_bytes: <0>
             primary_shard_count: <0>
                       doc_count: <0>
    primary_shard_size_avg_bytes: <0>
              primary_size_bytes: <0>
                total_size_bytes: <0>
               total_shard_count: <0>
    primary_shard_size_mad_bytes: <0>
                      node_count: <1>
                     index_count: <0>
                       enabled: <true>
                      data_hot: a map containing
 primary_shard_size_median_bytes: <0>
             primary_shard_count: <0>
                       doc_count: <0>
    primary_shard_size_avg_bytes: <0>
              primary_size_bytes: <0>
                total_size_bytes: <0>
               total_shard_count: <0>
    primary_shard_size_mad_bytes: <0>
                      node_count: <1>
                     index_count: <0>
                         slm: a map containing
                     available: <true>
                       enabled: <true>
                  policy_count: <0>
                  policy_stats: a map containing
                  retention_runs: <0>
                retention_failed: <0>
             retention_timed_out: <0>
         retention_deletion_time: "0s"
  retention_deletion_time_millis: <0>
           total_snapshots_taken: <0>
          total_snapshots_failed: <0>
         total_snapshots_deleted: <0>
total_snapshot_deletion_failures: <0>
                    policy_stats: an empty list
                 voting_only: a map containing
                     available: <true>
                       enabled: <true>
                data_streams: a map containing
                 indices_count: <0>
                     available: <true>
                  data_streams: <0>
                       enabled: <true>
              data_lifecycle: a map containing
                     lifecycle: expected a map but was <missing>
                     available: <true>
                       enabled: <true>
                         count: <unexpected> but was <0>
         default_rollover_used: <unexpected> but was <true>
                     retention: <unexpected> but was <{minimum_millis=0, maximum_millis=0, average_millis=0.0}>

  at __randomizedtesting.SeedInfo.seed([A2E9F58B9E81A215:2ABDCA51307DCFED]:0)
  at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:572)
  at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:524)
  at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
  at java.lang.reflect.Method.invoke(Method.java:578)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
  at java.lang.Thread.run(Thread.java:1623)

  Caused by: java.lang.AssertionError: Expected a map containing
                           eql: a map containing
                       available: <true>
                         enabled: <true>
                        features: a map containing
                             joins: a map containing
                  join_queries_three: <0>
                    join_queries_two: <0>
                          join_until: <0>
           join_queries_five_or_more: <0>
                   join_queries_four: <0>
                          sequence: <4>
                              keys: a map containing
                       join_keys_two: <0>
                       join_keys_one: <3>
                     join_keys_three: <0>
              join_keys_five_or_more: <0>
                      join_keys_four: <0>
                              join: <0>
                         sequences: a map containing
              sequence_queries_three: <1>
               sequence_queries_four: <0>
                sequence_queries_two: <3>
                      sequence_until: <0>
                    sequence_maxspan: <2>
       sequence_queries_five_or_more: <0>
                             event: <8>
                             pipes: a map containing
                           pipe_tail: <9>
                           pipe_head: <0>
                         queries: a map containing
                               all: a map containing
                               total: <12>
                              failed: <0>
                              _all: a map containing
                               total: <12>
                              failed: <0>
                frozen_indices: a map containing
                   indices_count: <0>
                       available: <true>
                         enabled: <true>
                           sql: a map containing
                        features: a map containing
                            having: <1>
                         subselect: <0>
                             limit: <1>
                           orderby: <7>
                             where: <3>
                              join: <0>
                           groupby: <1>
                           command: <0>
                             local: <0>
                       available: <true>
                         queries: a map containing
                              rest: a map containing
                               total: <10>
                              paging: <1>
                              failed: <0>
                               cli: a map containing
                               total: <0>
                              paging: <0>
                              failed: <0>
                            canvas: a map containing
                               total: <0>
                              paging: <0>
                              failed: <0>
                              odbc: a map containing
                               total: <0>
                              paging: <0>
                              failed: <0>
                              jdbc: a map containing
                               total: <0>
                              paging: <0>
                              failed: <0>
                            odbc32: a map containing
                               total: <0>
                              paging: <0>
                              failed: <0>
                            odbc64: a map containing
                               total: <0>
                              paging: <0>
                              failed: <0>
                              _all: a map containing
                               total: <10>
                              paging: <1>
                              failed: <0>
                         translate: a map containing
                               count: <1>
                         enabled: <true>
                        rollup: a map containing
                       available: <true>
                         enabled: <true>
          searchable_snapshots: a map containing
      shared_cache_indices_count: <0>
                   indices_count: <0>
                       available: <true>
         full_copy_indices_count: <0>
                         enabled: <true>
                     analytics: a map containing
                           stats: a map containing
                        rate_usage: <2>
                   normalize_usage: <0>
                     boxplot_usage: <3>
                 top_metrics_usage: <4>
                 multi_terms_usage: <1>
      cumulative_cardinality_usage: <0>
                string_stats_usage: <1>
          moving_percentiles_usage: <0>
                      t_test_usage: <1>
                       available: <true>
                         enabled: <true>
                      logstash: a map containing
                       available: <true>
                         enabled: <true>
                      security: a map containing
                       available: <true>
                         enabled: <false>
                     transform: a map containing
                       available: <true>
                         enabled: <true>
                           ccr: a map containing
      auto_follow_patterns_count: <0>
                       available: <true>
          follower_indices_count: <0>
                         enabled: <true>
                    health_api: a map containing
                       available: <true>
                         enabled: <true>
                        statuses: a map containing
                             green: <5>
                            values: a list containing
                                   0: "green"
                     invocations: a map containing
                             total: <5>
                      verbose_true: <5>
                       spatial: a map containing
                       available: <true>
                         enabled: <true>
                            ml: a map containing
                       inference: a map containing
                       deployments: a map containing
                             time_ms: a map containing
                                   avg: <0.0>
                               count: <0>
                   model_sizes_bytes: a map containing
                                 total: <0.0>
                                   min: <0.0>
                                   avg: <0.0>
                                   max: <0.0>
                    inference_counts: a map containing
                                 total: <0.0>
                                   min: <0.0>
                                   avg: <0.0>
                                   max: <0.0>
                    trained_models: a map containing
                    model_size_bytes: a map containing
                                 total: <0.0>
                                   min: <0.0>
                                   avg: <0.0>
                                   max: <0.0>
                estimated_operations: a map containing
                                 total: <0.0>
                                   min: <0.0>
                                   avg: <0.0>
                                   max: <0.0>
                               count: a map containing
                                 total: <1>
                           prepackaged: <1>
                                 other: <0>
                                _all: a map containing
                                 count: <1>
                 ingest_processors: a map containing
                                _all: a map containing
                             pipelines: a map containing
                                   count: <0>
                               time_ms: a map containing
                                     min: <0>
                                     max: <0>
                                     sum: <0>
                    num_docs_processed: a map containing
                                     min: <0>
                                     max: <0>
                                     sum: <0>
                          num_failures: a map containing
                                     min: <0>
                                     max: <0>
                                     sum: <0>
       data_frame_analytics_jobs: a map containing
                      memory_usage: a map containing
                    peak_usage_bytes: a map containing
                                 total: <0.0>
                                   min: <0.0>
                                   avg: <0.0>
                                   max: <0.0>
                              _all: a map containing
                               count: <0>
                   analysis_counts: an empty map
                            jobs: a map containing
                              _all: a map containing
                               count: <0>
                           detectors: a map containing
                                 total: <0.0>
                                   min: <0.0>
                                   avg: <0.0>
                                   max: <0.0>
                          created_by: an empty map
                          model_size: a map containing
                                 total: <0.0>
                                   min: <0.0>
                                   avg: <0.0>
                                   max: <0.0>
                           forecasts: a map containing
                                 total: <0>
                       forecasted_jobs: <0>
                       available: <true>
                       datafeeds: a map containing
                              _all: a map containing
                               count: <0>
                      node_count: <1>
                         enabled: <true>
                       watcher: a map containing
                       execution: a map containing
                           actions: a map containing
                                _all: a map containing
                                 total: <0>
                      total_time_in_ms: <0>
                           watch: a map containing
                             input: a map containing
                                _all: a map containing
                                 total: <0>
                                active: <0>
                           trigger: a map containing
                                _all: a map containing
                              ...
```

--- a/docs/reference/rest-api/usage.asciidoc
+++ b/docs/reference/rest-api/usage.asciidoc
@@ -331,14 +331,12 @@ GET /_xpack/usage
   "data_lifecycle" : {
     "available": true,
     "enabled": true,
-    "lifecycle": {
-        "count": 0,
-        "default_rollover_used": true,
-        "retention": {
-            "minimum_millis": 0,
-            "maximum_millis": 0,
-            "average_millis": 0.0
-        }
+    "count": 0,
+    "default_rollover_used": true,
+    "retention": {
+        "minimum_millis": 0,
+        "maximum_millis": 0,
+        "average_millis": 0.0
     }
   },
   "data_tiers" : {

